### PR TITLE
Replaced JAVA_BIN in openj9

### PR DESCRIPTION
**Pull request by LongyuZhang**
- replaced JAVA_BIN in openj9 test
[ci skip]

Signed-off-by: Longyu Zhang <longyu.zhang@ibm.com>

### CODE REVIEW

**Comment by LongyuZhang** [comment]
reviewers: @llxia @renfeiw

File: test/TestConfig/scripts/build_test.xml [hunk 1]
```diff
@@ -101,7 +100,7 @@
 
 
 	<target name="check_sdk">
-		<available file="${JAVA_BIN}" property="sdk.present" />
+		<available file="${TEST_JDK_HOME}" property="sdk.present" />
```
Comment (llxia) [inline test/TestConfig/scripts/build_test.xml]: I think `check_sdk ` target can be removed.

**Comment by llxia** [comment]
This PR cannot be merged until the internal reference to `check_sdk` target is removed.

**Comment by llxia** [comment]
jenkins test sanity,extended all jdk8

**Comment by llxia** [comment]
`TestAttachErrorHandling_SE80_0 ` failure in https://ci.eclipse.org/openj9/job/Test_openjdk8_j9_extended.functional_ppc64_aix_Personal/29/ is the same as https://github.com/eclipse/openj9/issues/3081

```
00:43:46.729  FAILED: test_startupShutdownRobustness
00:43:46.729  java.lang.AssertionError: target 17105066: wait loop not terminated
00:43:46.729  	at org.testng.AssertJUnit.fail(AssertJUnit.java:59)
00:43:46.729  	at org.testng.AssertJUnit.assertTrue(AssertJUnit.java:24)
00:43:46.729  	at org.testng.AssertJUnit.assertFalse(AssertJUnit.java:41)
```

**Comment by llxia** [comment]
jenkins test sanity,extended all jdk11

**Comment by llxia** [comment]
jenkins test sanity,extended all jdk11

**Comment by llxia** [comment]
jenkins test sanity,extended all jdk11

**Pull request merged**